ITEM.tsv

```diff
@@ -20,13 +20,13 @@ ITEM_20150317_000019	Due to its ease in manufacturing, this sword is frequently
 ITEM_20150317_000020	Cinquedea
 ITEM_20150317_000021	Recommended for those who desire a heavier and stronger sword.
 ITEM_20150317_000022	Sabre
-ITEM_20150317_000023	Primary weapon of the cavalry optimized for slashing. The Hilt also protects your hands.
+ITEM_20150317_000023	Primary weapon of the cavalry optimized for slashing. It features a guard on the hilt to protect the wielder's hands.
```

Comment:
Overly long translations can lead to errors like text breaking out of the text boxes in-game. Try to minimize text and not overcrowd information on items. "A guard on the hilt helps protect the user's hand."